### PR TITLE
#159 - refactors to be more canjs-like

### LINKED ISSUE #159: Get CanJS-like release script working
- include dist
- make sure travis is working
- get rid of grunt and bower
- make sure dist is exporting `FuncUnit`

### PR DESCRIPTION
**Pull request by mickmcgrath13**
Resolves #159 

* Refactor `package.json` scripts
* Remove grunt & bower
** Slight refactors to some of the tests to run after bower removal
* Tests seem to be all running, now, but the jasmine test seems wonky
* Docs still don't work
** This PR might not be where the fix for that goes.  It wasn't working before, either

### CODE REVIEW

File: test/adapters/jasmine.html [hunk 1]
```diff
@@ -2,23 +2,37 @@
...
+		<link rel="shortcut icon" type="image/png" href="../../node_modules/jasmine/node_modules/jasmine-core/images/jasmine_favicon.png">
+		<link rel="stylesheet" type="text/css" href="../../node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.css">
 	</head>
 
 	<body>
-		<script src="../../lib/jasmine/lib/jasmine-core/jasmine.js"></script>
-		<script src="../../lib/jasmine/lib/jasmine-core/jasmine-html.js"></script>
+		<script src="../../node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.js"></script>
+		<script src="../../node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine-html.js"></script>
+		<script src="../../node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/boot.js"></script>
```
Comment (mickmcgrath13) [inline test/adapters/jasmine.html]: This stuff is ugly, but I wasn't concerned with doing this the best way.  It used to grab these files from Bower.  This should probably be updated to be more correct.

File: test/drag/drag.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
-//load a lot of stuff ...
-steal(function(){
+steal("../../browser/jquery", "../../node_modules/jquerypp/index", function($){
```
Comment (mickmcgrath13) [inline test/drag/drag.js]: It'd probably be better to "require" these files so we're more consistent with other files.

File: test/adapters/jasmine.html [hunk 3]
```diff
@@ -2,23 +2,37 @@
...
-				var htmlReporter = new jasmine.HtmlReporter();
+
+				var htmlReporter = new jasmine.HtmlReporter({
+			    env: jasmineEnv,
+			    queryString: queryString,
+			    onRaiseExceptionsClick: function() { queryString.setParam("catch", !env.catchingExceptions()); },
+			    getContainer: function() { return document.body; },
+			    createElement: function() { return document.createElement.apply(document, arguments); },
+			    createTextNode: function() { return document.createTextNode.apply(document, arguments); },
+			    timer: new jasmine.Timer()
```
Comment (mickmcgrath13) [inline test/adapters/jasmine.html]: The HTMLReporter object was just copied from [Jasmine's docs](http://jasmine.github.io/2.0/boot.html#section-Reporters).  I didn't look at the logic;  I was just trying to get this refactor to be in a solid state.

**Comment by mickmcgrath13** [comment]
Weird... So, it looks like the tests aren't passing in Travis:
image

...But that particular test passed for me locally:
image

File: test/adapters/jasmine.html (hunk 1 above)
Comment (matthewp) [inline test/adapters/jasmine.html]: looks fine to me.

File: test/drag/drag.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
+	window.jQuery = $;
```
Comment (matthewp) [inline test/drag/drag.js]: Why is this needed? Just a leftover from debugging?

Comment (mickmcgrath13) [inline test/drag/drag.js]: It's not needed.  There was some copy/paste/debugging going on.  I'll remove it.

Comment (mickmcgrath13) [inline test/drag/drag.js]: Weeelll... When I said it's not needed, I guess I was a littler overly confident.  That test breaks without it, so it *is* needed.  Why it breaks, though, is not clear to me..
